addons/packetfence-perl/install_cpan.sh

```diff
@@ -96,10 +177,11 @@ OLDIFS=$IFS
 IFS=','
 while read cpanName cpanVersion cpanInstall cpanTest cpanAll
 do
-  ListCsvModInstall+=( $cpanInstall )
   ListCsvModName+=( $cpanName )
+  ListCsvModVersion+=( $cpanVersion )
```

Comment:
To be more consistant this one `ListCsvModVersion` should be declared with the others to be more readable. Like this:
```
ListCsvModInstall=()
ListCsvModName=()
ListCsvModTest=()
ListCsvModVersion=()
```
(172-174)